ejb3/src/main/java/org/jboss/as/ejb3/timerservice/persistence/filestore/EjbTimerXmlParser_1_0.java

```diff
@@ -155,9 +153,6 @@ private void parseTimer(XMLExtendedStreamReader reader, List<TimerImpl> timers)
                         if (loadableElements.info != null) {
                             builder.setInfo((Serializable) deserialize(loadableElements.info));
                         }
-                        if (loadableElements.primaryKey != null) {
```

Comment:
Are these changes going to result in previously working configurations now failing?

As this is a versioned parser is there also a schema to back it up, does there need to be a version 1.1 or are there already later versions with this removed?

Reply:
The changes here are not structural changes so should still be compatible with existing config. All changes are internal implementation, and if any persisted timers contain primary key field, it will just be ignored.